### PR TITLE
stolostron/backlog#26070--post GA changes that had to wait bc of build

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

**Review by oafischer — APPROVED** [review]

File: clusters/hosted_control_planes/configure_hosted.adoc [hunk 1]
```diff
@@ -314,4 +314,4 @@ api-int.example.krnl.es.    IN A 192.168.122.22
 *.apps.example.krnl.es. IN A 192.168.122.23
 ----
 
-You can now deploy the SR-IOV Operator. See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/networking/hardware-networks#sriov-operator-hosted-control-planes_configuring-sriov-operator[Deploying the SR-IOV Operator for hosted control planes] to learn more about deploying the SR-IOV Operator.
+You can now deploy the SR-IOV Operator. See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/networking/hardware-networks#sriov-operator-hosted-control-planes_configuring-sriov-operator[Deploying the SR-IOV Operator for hosted control planes] to learn more about deploying the SR-IOV Operator.
```
Comment (dockerymick) [inline clusters/hosted_control_planes/configure_hosted.adoc:317]: broken link for the DNS reference 
Screen Shot 2023-02-09 at 11 59 18 AM

ignore the fact that the tool lists 2 broken links. The tools counts **Code** tab as a broken link. The DNS link is the only one that is broken

Comment (dockerymick) [reply clusters/hosted_control_planes/configure_hosted.adoc:317]: Screen Shot 2023-02-09 at 12 03 59 PM

File: clusters/hosted_control_planes/create_hosted.adoc [hunk 2]
```diff
@@ -250,7 +249,6 @@ hypershift create cluster agent \
     --agent-namespace=${HOSTED_CONTROL_PLANE_NAMESPACE} \
     --base-domain=${BASEDOMAIN} \
```
Comment (dockerymick) [inline clusters/hosted_control_planes/create_hosted.adoc:250]: link to Configuring hosted control planes (Technology Preview) is broken. I will take a look

**Comment by swopebe** [comment]
Adel approved on Slack.

**Comment by swopebe** [comment]
Thanks for the link work @dockerymick

**Pull request merged**